### PR TITLE
Update docker.md, domserver.md and judgehost.md

### PR DESCRIPTION
**Pull request by 87ouo**
Changed the address of Chinese mainland`s docker-hub mirror source to speed up the progress of pulling docker images in Chinese mainland area. And updated some grammar formats in docker.md

### CODE REVIEW

**Comment by 87ouo** [comment]
Update the environment of domserver.md and judgehost.md to DOMjudge 7.3.3 and Ubuntu 20.04 LTS / 21.04 .

**Comment by yang-er** [comment]
Please resolve the conflicts first. You may have to cherry pick 322d7d4...bee5c13 here.

**Comment by 87ouo** [comment]
It seems OK now.

File: judgehost.md [hunk 1]
```diff
@@ -49,13 +51,13 @@ make judgehost && sudo make install-judgehost
 ### 添加用户
 
 ```shell
-useradd -d /nonexistent -U -M -s /bin/false domjudge-run
-# 如果 judgehost 拥有多个 CPU 核心，你可以添加额外的用户来支持绑定
+sudo useradd -d /nonexistent -U -M -s /bin/false domjudge-run
+# 如果 judgehost 拥有多个 CPU 核心，你可以添加额外的用户来支持绑定，但是要记得修改 judgehost-static.php 等文件里的 `RUNUSER` 。
```
Comment (yang-er) [inline judgehost.md]: Why modify `RUNUSER`?

Comment (87ouo) [reply judgehost.md]: My apologies. I was misunderstood the original author`s meaning. It is a WRONG step. I have already fix it in the newest PR. Thank you for your thoughtfulness.

**Pull request merged**